src/plugins/discover/public/application/helpers/get_context_url.test.ts

```diff
@@ -12,19 +12,32 @@ const filterManager = ({
   getGlobalFilters: () => [],
   getAppFilters: () => [],
 } as unknown) as FilterManager;
+const addBasePath = (path: string) => `/base/${path}`;
```

Comment:
Nit: `/base/${path}` -> `/base${path}` and the expected url would be `/base/app/....`